_component_design/switches.md

```diff
@@ -14,72 +14,91 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-<!--{% include todo.html content="Update focus states, expand on inline validation, and improve
-keyboard accessibility." %}-->
 
-The switch should only be used in **mobile** designs. Enabling preferences or
-settings is a prime use case for this component. The switch can be used with or
-without text, depending on the context provided in the switch label. If more
-complex text is needed in the switch than yes/no or on/off, use a **toggle**.
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-## Default
+### When to use
+{: .hxSubSectionTitle}
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/selectors-switch.png){:width="30%"}
+Use a switch if the user is turning something on or off, or saying yes or no to a question. There is no neutral option; the answer is either set to on or off. Enabling preferences or settings is a prime use case for this component. The switch should always be used with text. 
```

Comment:
You can probably remove "There is no neutral option; the answer is either set to on or off." - assigning 'yes or no' options implicitly states that there is no neutral option.

Reply:
I'm not sure at all what you're trying to say next with, "Enabling preferences or settings is a prime use case for this component. The switch should always be used with text." 
Do you mean that you cannot just have a switch without actually stating that one side is yes, and one side is no?

Reply:
Now reads, "“Use a switch if the user is turning something on or off, or saying yes or no to a question; there is no neutral option. Enabling preferences or settings is a prime use case for this component, explicitly stating the current status with text.”